src/entities/session/api/sessionApi.ts

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from "@tanstack/react-query";
```

Comment:
you wanted to remove react-query
why is it here again?

Reply:
![image](https://github.com/erastian/REACT_news_client/assets/15855781/7ae3a22c-58bd-489c-ae20-6f234a51708b)